.travis.yml

```diff
@@ -26,8 +26,10 @@ before_script:
   - "export EVSS_SAMPLE_CLAIMANT_USER='{\"uuid\": \"1234\", \"first_name\": \"Jane\",\"last_name\": \"Doe\",\"edipi\": \"1105051936\",\"participant_id\": \"123456789\"}'"
   - export MVI_URL='http://ps-dev.commserv.healthevet.va.gov:8110/psim_webservice/IdMWebService'
   - export EVSS_S3_UPLOADS=false
-  - export VHA_MAPSERVER_URL='https://maps.va.gov/server/rest/services/PROJECTS/Facility_Locator/MapServer'
+  - export VHA_MAPSERVER_URL='https://services3.arcgis.com/aqgBd3l68G8hEFFE/ArcGIS/rest/services/VHA_Facilities/FeatureServer'
   - export VHA_MAPSERVER_LAYER='0'
+  - export NCA_MAPSERVER_URL='https://services3.arcgis.com/aqgBd3l68G8hEFFE/ArcGIS/rest/services/NCA_Facilities/FeatureServer'
```

Comment:
Up until recently it was not clear whether the different services were going to share a base URL or not. I'm still not 100% credulous that e.g. the "NCA_Facilities" part of the path is going to remain stable so I'm hesitant to put it in code vs. config. I am likely to eliminate the separate "LAYER" env vars in the future by switching to a standard HTTP client instead of the geoservices gem which doesn't add much value as it's just assembly a query string.